api-schema/xdb.yaml

```diff
@@ -499,6 +510,22 @@ components:
           $ref: '#/components/schemas/RetryPolicy'
         executeApiRetryPolicy:
           $ref: '#/components/schemas/RetryPolicy'
+        stateFailureRecoveryInfo:
```

Comment:
it maybe nice to rename this to `stateFailureRecoveryPolicy`

Reply:
Also let's create a separate definition for `StateFailureRecoveryOptions` and use `$ref`